src/providers/bitpay-card/bitpay-card.ts

```diff
@@ -491,13 +495,24 @@ export class BitPayCardProvider {
   }
 
   public register() {
+    this.globalization
+      .getLocaleName()
+      .then(res => {
+        this.countryCode = res.value.split('-')[1];
+      })
+      .catch(err => {
+        this.logger.debug('Error getting country code', err);
+      });
+
     this.isActive(isActive => {
       this.homeIntegrationsProvider.register({
         name: 'debitcard',
         title: 'BitPay Visa® Card',
         icon: 'assets/img/bitpay-card/icon-bitpay.svg',
         page: 'BitPayCardIntroPage',
-        show: !!this.configProvider.get().showIntegration['debitcard'],
+        show:
+          !!this.configProvider.get().showIntegration['debitcard'] &&
+          this.countryCode === 'US',
```

Comment:
Getting a value from an async function could not work in this case. I think `this.isActive` function should be inside the scope of `getLocalName`.